test/unit/test_anal_function.c

```diff
@@ -70,9 +70,63 @@ bool test_r_anal_function_relocate() {
 	mu_end;
 }
 
+bool test_r_anal_function_labels() {
+	RAnal *anal = r_anal_new ();
+
+	RAnalFunction *f = r_anal_create_function (anal, "do_something", 0x1337, 0, NULL);
+
+	bool s = r_anal_function_set_label (f, "smartfriend", 0x1339);
+	mu_assert ("set label", s);
```

Comment:
may i suggest to use `mu_assert_true`? (note `mu_assert_true` has the actual value as first arg, like all the other `mu_assert_*` macros.